DigitalLearningSolutions.Web/Views/Supervisor/RemoveConfirm.cshtml

```diff
@@ -25,12 +25,30 @@
       </div>
     </div>
     <p class="nhsuk-body-m">You are supervising @Model.CandidateAssessmentCount profile assessments for this member of staff.</p>
-    <p class="nhsuk-body-m">Are you sure that you wish to remove them from your staff list?</p>
-    <a class="nhsuk-button nhsuk-button--danger nhsuk-u-margin-top-4" asp-action="RemoveSupervisorDelegate" asp-route-supervisorDelegateId="@Model.ID">
-      Remove
-    </a>
-    <a class="nhsuk-button nhsuk-button--secondary nhsuk-u-margin-top-4" asp-action="MyStaffList">
-     Cancel
-    </a>
+    <form method="post" asp-controller="Supervisor">
+      <div class="nhsuk-checkboxes__item">
+        <nhs-form-group nhs-validation-for="ConfirmedRemove">
+          @Html.CheckBoxFor(m => m.ConfirmedRemove, new { @class = "nhsuk-checkboxes__input", required = "required" })
```

Comment:
Rather than validating this using **required** in the HTML checkbox input tag, we should validate using the NHS Design System approach used in Web/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml. 

This is a more complex implementation but we need to stick with NHS Design System validation particularly in our high usage areas of the site to make the user experience consistent.